app/src/main/java/org/mifos/mobile/ui/fragments/HelpFragment.kt

```diff
@@ -121,12 +129,25 @@ import javax.inject.Inject
             }
 
             override fun onQueryTextChange(newText: String): Boolean {
-                faqAdapter?.updateList(presenter?.filterList(faqArrayList, newText))
+                val filteredList = presenter?.filterList(faqArrayList, newText)
+                filteredList?.let {
+                    if (it.isNotEmpty()) {
+                        sweetUIErrorHandler.hideSweetErrorLayoutUI(rvFaq, layoutError)
+                        faqAdapter?.updateList(it)
+                    } else {
+                        showEmptyUI()
+                    }
+                } ?: showEmptyUI()
                 return false
             }
         })
     }
 
+    private fun showEmptyUI() {
```

Comment:
Let's use a better name for this `method/function`, it appears to be too generic. for instance `showEmptyFAQUI` or something better